olm-catalog/serverless-operator/project.yaml

```diff
@@ -31,14 +31,14 @@ requirements:
         max: '4.15'
         label: 'v4.12'
 dependencies:
-    serving: knative-v1.12
+    serving: knative-v1.14
```

Comment:
Do we need a mechanism to ignore this gap 1.12 -> 1.14 anywhere? What is the status with version checking?

Reply:
Should be fine (not tested). We removed the semver check in S-O.